src/containers/language/language.js

```diff
@@ -14,10 +14,14 @@ const languageInLocalStorage = getFromLocalStorage('language') || 0;
 const Language = () => {
   const styles = useStyles();
   const dispatch = useDispatch();
+
   dispatch(changeLanguage(languageInLocalStorage));
 
   const handleChange = (e) => {
     const targetValue = e.target.value;
+    if (targetValue === undefined) {
```

Comment:
I think it is unnecessary to use === here. try to change it  if (targetValue){...} it should include all false values